src/commands/base-command.ts

```diff
@@ -148,6 +148,15 @@ async function getRepositoryRoot(cwd?: string): Promise<string | undefined> {
   }
 }
 
+export type BaseOptionValues = {
```

Comment:
`json` is a weird one IIRC. I moved it out of BaseCommand but it's still referenced in a generic context in a couple places. I guess we'll gross that bridge when we get there.